### PR TITLE
Update french translations

### PR DESCRIPTION
**Pull request by zayfix**
Hello ! I'm back after almost [2 years](https://github.com/official-antistasi-community/A3-Antistasi/pull/1109) for more french translations !

With the amount of translation needed, i might i've missed a few things, don't hesitate to correct me.

By the way i just wanted to say you guys are doing an amazing job !

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information: Updated french translations.
    

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
- Load up Antistasi with the game language set to French.

********************************************************
Notes: N/A

### CODE REVIEW

**Comment by Bob-Murphy** [comment]
Hey, welcome back and thank you for the commit.
I'll see to get it integrated.

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]

File: A3A/addons/core/Stringtable.xml [hunk 1]
```diff
@@ -2192,919 +2192,1097 @@
...
         <Original>Recruit Mortar Team</Original>
+        <French>Recruter Équipe de mortier</French>
         <Chinesesimp>招募迫击炮小组</Chinesesimp>
         <Russian>Артилерия</Russian>
       </Key>
     </Container>
     <Container name="player_money">
       <Key ID="STR_antistasi_dialogs_player_money">
         <Original>Player and Money Interaction</Original>
+        <French>Player and Money Interaction</French>
```
Comment (Eli-Porter) [inline A3A/addons/core/Stringtable.xml:3015]: Change it to either "Interaction du Joueur et de l'Argent" or "Interaction Joueur et Argent"

**Review by Eli-Porter — CHANGES_REQUESTED** [review]

**Comment by Bob-Murphy** [comment]
Sorry for the delay.
Thank you Eli for checking and Thank you Zayfix for the changes.
I try to get my build bipeline up (new computer) so I can test and confirm it working this weekend.

**Review by Bob-Murphy — APPROVED** [review]
Läuft

**Comment by Bob-Murphy** [comment]
@zayfix I finally got time to check it and it's good and merged for next update.
Thank you!

**Pull request merged**